pkg/virtual/workspaces/auth/cache.go

```diff
@@ -224,14 +224,13 @@ func NewAuthorizationCache(
 	ac.lastSyncResourceVersioner = workspaceLastSyncResourceVersioner
 	ac.syncHandler = ac.syncRequest
 	ac.rwMutex = sync.RWMutex{}
-	return ac
+	return &ac
 }
 
 // Run begins watching and synchronizing the cache
-func (ac *AuthorizationCache) Run(period time.Duration) {
+func (ac *AuthorizationCache) Run(period time.Duration, stopCh <-chan struct{}) {
```

Comment:
why no context?

Reply:
I don't see why we would need a context here. Just need a stop channel.

Reply:
And this way it is consistent with what is already done in the `WorkspaceCache`: https://github.com/kcp-dev/kcp/blob/main/pkg/virtual/workspaces/cache/cache.go#L94